### ISSUE TITLE
entering dates for wild caught animals that were in the lab for months before euth

### ISSUE THREAD

**Issue by acoconis** [ArctosDB/arctos#2415]
What date should be entered for specimen event and collecting event if the animal was wild caught but then in the lab on diet trials for months before euthanization? Thanks!

**Comment by dustymc** [comment]
I think "most correct" is two specimen events (perhaps with a new value in http://arctos.database.museum/info/ctDocumentation.cfm?table=CTSPECIMEN_EVENT_TYPE). "Normal" is the date of capture as the single specimen event and attribute [verbatim preservation date](http://arctos.database.museum/info/ctDocumentation.cfm?table=CTATTRIBUTE_TYPE&field=verbatim%20preservation%20date) for kill date.

**Comment by campmlc** [comment]
You can also use verbatim preservation date attribute.

**Comment by Jegelewicz** [comment]
> I think "most correct" is two specimen events (perhaps with a new value in http://arctos.database.museum/info/ctDocumentation.cfm?table=CTSPECIMEN_EVENT_TYPE).

This would be the best way to do this although we have hashed over the "new" specimen event type before and eventually ended up using the "verbatim preservation date" option instead.

See ArctosDB/arctos#1502 can we add an event type "preservation"?

**Comment by acoconis** [comment]
As mentioned in your discussion in ArctosDB/arctos#1502: "The disadvantage would be these events/localities are not very useful to most research. If they could be excluded from the locality search unless specifically requested it might fix that issue. Maybe we create a new verification status?"

I didn't think of this at first, but this could be a real issue. I'm not sure the verification status would be obvious to most users to filter when searching. Maybe though...

**Comment by dustymc** [comment]
>not very useful to most research.

Possibly, but when so it's likely critical - captivity is likely to significantly influence weight, size, growth rate, blood chemistry, etc.

>verification status

That comment may be in regard to an older model. This definitely isn't verification status, but could (should?) be specimen event type.

>obvious

The attribute approach would look about the same in any context. An additional event would be represented in various ways, including packaged as an Occurrence. I'd probably notice events before attributes, but I'm not sure that's true for anyone else.

There's also the matter of what the various data containers can hold. Additional events can get at places, date ranges, etc., where the attribute is more limited.

**Comment by acoconis** [comment]
Yes, this all makes sense. It is critical that we document captivity because it absolutely matters for morphology. However, I should have been more clear about the concern about locality. If we create a new specimen event for when the animal died (in captivity), we'd have to include locality information (correct?), and in doing so, it may be confusing to those using the locality information for say something like building species distribution models.

**Comment by dustymc** [comment]
>we'd have to include locality information

Yes, but "we're not saying" is always available. (Not something I'd recommend, but there's some flavor of "we don't know" available for every required concept.)

>species distribution models

Anyone doing that with Arctos data and not looking at the metadata is just going to end up with garbage. I'm certainly always happy to discuss how to make that more obvious, but Arctos is well beyond a simple "one true thing" model and contains lots of data which aren't suitable for just about any given purpose.

**Comment by acoconis** [comment]
Ah okay. I didn't realize "we're not saying" was an option. 

And you're right about the metadata comment, although I'm sure it happens. I was just trying to think about applications in which it may be confusing.

In this case, fwiw, my vote is for 2 specimen entries with no locality information for the captive entry. And a new specimen event type - maybe "preservation".

**Comment by dustymc** [comment]
http://handbook.arctosdb.org/documentation/higher-geography.html#continent--ocean
http://arctos.database.museum/geography.cfm?higher_geog=recorded

**Comment by acoconis** [comment]
Hi all! I just wanted to check in about this conversation again. I'd like to get my specimens at UNR in order and move forward with entering more (that have also been wild caught but spent some time in lab feeding trials). 

Should I just add the "verbatim preservation date" attribute for now and then switch to entering 2 events if/when that is decided? Again, I think we would need an additional event type added to the form (maybe "preservation"?)

I would happy to write up a "how-to" if there is a consensus.

Thanks!

**Comment by campmlc** [comment]
That sounds reasonable to me.

**Comment by Jegelewicz** [comment]
> Should I just add the "verbatim preservation date" attribute for now and then switch to entering 2 events if/when that is decided? 

That's what I did with all the UTEP stuff - so you would be consistent with them. Sounds good!

**Comment by acoconis** [comment]
Great thanks!

**Comment by dustymc** [comment]
See also https://github.com/ArctosDB/arctos/issues/1502
